app/controllers/friendships_controller.rb

```diff
@@ -44,6 +44,18 @@ def create
   # PATCH/PUT /friendships/1 or /friendships/1.json
   def update
     respond_to do |format|
+      if Friendship.where(buddy_id: current_user.id, status: :accepted).length == 0
+        if @friendship.update(friendship_params)
+          format.html do
+            redirect_to @friendship, notice: "It's dangerous to go alone, take this Friend."
```

Comment:
I thought that but testing it in the browser it only actually works when length === 0 🤔 

Why do you think that it should be length 1? My thought process was since it's for the first friendship and the update is happening just after in the if `@friendship.update` 

The message can be swapped out no problem, I agree it will make it cleaner.

Reply:
Well this PR was a while ago.... I was thinking we'd check the length inside the format.html and swap between two strings there. But we could also use a variable from outside and check against 0.